appendix-basic65.tex

```diff
@@ -7225,9 +7221,9 @@ \subsection{RMOUSE}
 \item [Format:] {\bf RMOUSE xvar, yvar, butvar}
 \item [Usage:] Reads mouse position and button status.
 
-               {\bf xvar} numerical variable receiving x-position.
+               {\bf xvar} numerical variable where the x-position will be stored.
```

Comment:
Hmm, I let it go the first read through, but the more I see this term `numerical`, the more it seems to bug me. Would plain old `numeric` be a better fit? (i.e., `numeric variable`?). Will leave it up to you.

Reply:
it's a 50/50 for me, i'll change it to numeric